dod-wrapper/src/main/java/com/revolut/rxdata/dod_wrapper/DataObservableDelegateWrapperDispatchers.kt

```diff
@@ -33,4 +33,10 @@ internal object DataObservableDelegateWrapperDispatchers {
         IO = dispatcher
         Unconfined = dispatcher
     }
-}
\ No newline at end of file
+
+    @TestOnly
+    fun resetDispatchers() {
+        IO = Dispatchers.IO
+        Unconfined = Dispatchers.Unconfined
```

Comment:
I'd reuse this method in init phase as well, not repeat IO = Dispatchers.IO / Unconfined = Dispatchers.Unconfined twice

Reply:
We'll need to declare the properties as `lateinit`, otherwise compiler is not satisfied. Do you think it's worth it?

Reply:
sounds fair, unless you see any issues with that

Reply:
Done 😄